styleguide/derek/_partials/solutions/header-smallIMAC.ejs

```diff
@@ -0,0 +1,13 @@
+<!--Header Short-->
+<div class="imacBanner small">
+  <div class="imacBanner-container">
+    <div class="imacBanner-textRow">
+      <div class="imacBanner-textContainer imacBanner-headlineSmall">
+        <h2 class="imacBanner-headline">the cloud runs on its own,<br/>with lots of help</h2>
```

Comment:
Fixed the captilization but the punctuation has been brought to brands attention so we will see. For now this is what I was given for headlines unfortunately.